CHARTER.md

```diff
@@ -1,28 +1,25 @@
 # Open API designer project charter
 
 ## Objective
-The goal is to have an OpenAPI design tool and a storage backend that can be integrated with APInf platform. We will host the on our  servers and we are open if anyone want to use or build their own interface above it. Whoever uses our Open API Designer service to design their APIs, we offer them free cataloging at https://apinf.io 
+The goal is to have an OpenAPI design tool that can be integrated with APInf platform and storage backends.
 
 ## Risks
-* Duplication of effort
 * Misalignment with business or community goals
 * Failure to execute or prioritize properly
 * Miscommunication of project intent or scope
-* Going into competition with other companies and solutions
 
 ## Value proposition
-User can create an OpenAPI document without knowing the specification or being familiar with JSON or YAML. User is guided by user interface elements such as hints in form fields and mandatory fields.
+User can create and edit OpenAPI documents without knowing the specification or being familiar with JSON or YAML. User is guided by user interface elements such as hints in form fields and mandatory fields and by the sequence of the form.
```

Comment:
> by the sequence of the form.

Consider rewording as:

User is guided by user interface elements, such as hints in form fields and mandatory fields, as well as the user flow. 

User flow means more than 'the order of the field', and can include multi-page editing experience. This also means that the views can be reordered, so they do not so closely align with the OpenAPI specification.